doc/classes/ProjectSettings.xml

```diff
@@ -2303,6 +2306,9 @@
 		<member name="physics/3d/run_on_separate_thread" type="bool" setter="" getter="" default="false">
 			If [code]true[/code], the 3D physics server runs on a separate thread, making better use of multi-core CPUs. If [code]false[/code], the 3D physics server runs on the main thread. Running the physics server on a separate thread can increase performance, but restricts API access to only physics process.
 		</member>
+		<member name="physics/3d/run_on_separate_thread.editor" type="bool" setter="" getter="" default="false">
+			Editor override for [member physics/3d/run_on_separate_thread] to specify whether the editor's 3D physics server should run on a separate thread or not.
```

Comment:
Ditto

Reply:
Added: 
![image](https://github.com/user-attachments/assets/a08f4875-4b0c-4a51-8754-0f795dc90380)

This kind of inline with what you were thinking? 🤔

Reply:
Yes but not quite. There's no need to explain what the setting override is. There does need to be an explanation as to why this is disabled by default in the editor.